ops/lib.rs

```diff
@@ -157,14 +157,18 @@ fn codegen_v8_async(
   margs: MacroArgs,
   asyncness: bool,
 ) -> TokenStream2 {
-  let arg0 = f.sig.inputs.first();
-  let uses_opstate = arg0.map(is_rc_refcell_opstate).unwrap_or_default();
-  let args_head = if uses_opstate {
-    quote! { state, }
-  } else {
-    quote! {}
-  };
-  let rust_i0 = if uses_opstate { 1 } else { 0 };
+  let MacroArgs { is_v8, .. } = margs;
+  let special_args = f
+    .sig
+    .inputs
+    .iter()
+    .map_while(|a| {
+      (if is_v8 { scope_arg(a) } else { None }).or_else(|| opstate_arg(a))
```

Comment:
`if is_v8 { scope_arg(a) } else { opstate_arg(a) }` possibly?

Reply:
That's not something I changed – it's there on `master`.